commands/react_hgs/consts.py

```diff
@@ -14,15 +14,20 @@
     "<:UATroll:1142824293470838854>",
 ]
 
-KEYWORDS_EN = (
+KEYWORDS_EN = [
     "HGS",
     "hot gay sex",
-)
+]
+
+KEYWORDS_LT = [r"kar[šs]tas\s+g[ėe]j[ųu]\s+seksas"]
+
+KEYWORDS_LV = [r"(?:Dedzīgs|Kaislīgs)\s+geju\s+sekss"]
 
-KEYWORDS_TW = (r"(?:同志)(?:激情|熱烈)(?:性愛|性交)", r"ㄏㄍㄙ")
+KEYWORDS_UA = [r"гарячий\s+гей\s+секс"]
 
+KEYWORDS_TW = [r"(?:激情|熱烈)(?:同志|同性)(?:性愛|性交)", r"ㄏㄍㄙ"]
 
 HGS_REGEX = compile(
-    rf"\b(?:{'|'.join(KEYWORDS_EN)})\b|{'|'.join(KEYWORDS_TW)}",
+    rf"\b(?:{'|'.join(KEYWORDS_EN + KEYWORDS_LT + KEYWORDS_LV + KEYWORDS_UA)})\b|{'|'.join(KEYWORDS_TW)}",

```

Comment:
instead of having different lists per language you can probably make a dict of hgs

Reply:
coding is hard

Reply:
beep boop

Reply:
I think we don't need to because we don't use it anywhere else and also the TW regex is different (because of the `\b`) so we would need to take that into account